kitchen.dokken.yml

```diff
@@ -1,8 +1,9 @@
 driver:
   name: dokken
   privileged: true  # because Docker and SystemD/Upstart
-  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
+  chef_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
   chef_license: accept-no-persist
+  clean_dokken_sandbox: true
```

Comment:
Please remove this.

Reply:
Why? This does not appear to affect the testing environment in Github Actions; however, I've found that having it in place helps with local development, as it keeps the development environment clean while testing/developing as near as possible to the release environment.

Reply:
This hasn't been a standard in our files and I'd rather have it in a state that's similar to other cookbooks. If it's useful locally, then you always use `.kitchen.local.yml` for those types of settings.

Reply:
KITCHEN_LOCAL_YAML is already set to the `kitchen.dokken.yml` file for this repo -- Test Kitchen will ignore `.kitchen.local.yml` and `kitchen.local.yml`.